Lesson06/HW06.ThirdArrayCreator/HW06.ThirdArrayCreator/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace HW06.ThirdArrayCreator
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("enter a number:");
+            int n = Convert.ToInt32(Console.ReadLine());
+            for (int i = 1; i <= n; i++)
+            {
+                for (int x = i; x <= n; x++)
+                {
+                    Console.Write(" ");
+                }
+                for (int j = 1; j <= i; j++)
+                {
```

Comment:
OMG. The solution is not from this task, and even if, it is still incorrect.

Reply:
done